README.md

```diff
@@ -779,6 +779,43 @@ if err != nil {
 }
 ```
 
+### Utils for your end to end (e2e) tests and integration tests
+
+To ease your e2e tests, we exposed dedicated management methods,
+that way, you won't need to use 3rd party messaging services to get the actual sign-in/up email or sms.
```

Comment:
It is not the email/sms
It is the token/code without the need of parsing the email/sms
Maybe consult with Itai on the wording

Reply:
It is email/sms.
Shriki approved already.